Documentation/ExtensionArchitecture/FileStructure/ExtConfTemplate.rst

```diff
@@ -142,7 +136,6 @@ You can also define nested options using the TypoScript notation:
 This will result in a multidimensional array:
 
 .. code-block:: php
-   :caption: EXT:some_extension/Classes/SomeClass.php
```

Comment:
We once made the rule that all Code Snippets should have a caption that helps new users understand where such a code Snippet should be put. I agree that "SomeClass" is not much information but at least you can see it is a PHP file in the Classes Folder of an Extension.

Reply:
I disagree. It is just a random output of some PHP array, so no real PHP code. And therefore, no code to put into some class.

Reply:
I such a case we label it as exactly that: Output of array xy

Reply:
is it the content of `$myConfiguration = $this->extensionConfiguration->get('my_extension_key')`? also now that I look at it, it is not valid PHP at all